src/session/session_helper.c

```diff
@@ -134,3 +134,35 @@ __wt_session_dump(WT_SESSION_IMPL *session, WT_SESSION_IMPL *dump_session, bool
     __wt_scr_free(session, &buf);
     return (ret);
 }
+
+/*
+ * __wt_session_set_last_error --
+ *     Stores information about the last error to occur during this session.
+ */
+int
+__wt_session_set_last_error(
+  WT_SESSION *wt_session, int err, int sub_level_err, const char *err_msg_content)
```

Comment:
no need wt_session, can just call it `session` instead. I believe wt_session naming convention should only be user-facing

Reply:
Done in [Pass WT_SESSION_IMPL into __wt_session_set_last_error instead of WT_S…](https://github.com/wiredtiger/wiredtiger/pull/11361/commits/6fe3e3c5ccc5ba957a2fb8dab7b7992243af183b)